app/models/admin_dashboard_data.rb

```diff
@@ -51,18 +25,16 @@ def as_json(_options = nil)
 
   def problems
     problems = []
-    self.class.problem_syms.each do |sym|
-      message = public_send(sym)
-      problems << Problem.new(message) if message.present?
-    end
     self.class.problem_blocks.each do |blk|
       message = instance_exec(&blk)
-      problems << Problem.new(message) if message.present?
+      problems << ProblemCheck::Problem.new(message) if message.present?
     end
     self.class.problem_messages.each do |i18n_key|
       message = self.class.problem_message_check(i18n_key)
-      problems << Problem.new(message) if message.present?
+      problems << ProblemCheck::Problem.new(message) if message.present?
     end
+    problems.concat(ProblemCheck.realtime.flat_map { |c| c.call(@opts).map(&:to_h) })
```

Comment:
This is a critical line. It moves the entry point from `@@problem_syms` to `ProblemCheck.realtime`.